app/src/services/read-only.service.js

```diff
@@ -0,0 +1,29 @@
+module.exports = class ReadOnlyService {
+
+    constructor(blacklist = [], whitelist = []) {
+        this.blacklist = blacklist;
+        this.whitelist = whitelist;
+    }
+
+    isBlacklisted(path) {
+        return this.blacklist.some((element) => new RegExp(element).test(path));
```

Comment:
regexes kill puppies. Let's use an array of objects, where we match verb and url (including potential variables, with the :var notatioin)

Reply:
Going to add support for the `:var` notation, it is definitely going to be needed :ok_hand: 

About regexes, I think that at least for now I am going to keep supporting them - we have 12 `find-by-ids` endpoints that would definitely benefit from having regex support :)

However, I am going to remove regexes from the docs, since they can be used as a hack in a rush situation, but it is not something that we want to be used by default when using this read-only mode :+1:

Reply:
After some thought, the `:var` notation would only make sense to implement if we removed support for regex, which is not the case. So keeping this as-is for now.